temboardagent/plugins/dashboard/db.py

```diff
@@ -0,0 +1,60 @@
+# coding: utf-8
+
+import json
+import os
+import sqlite3
+from textwrap import dedent
+
+
+def bootstrap(path, dbname):
+    """Create SQLite3 database model to store stuff."""
+
+    with sqlite3.connect(os.path.join(path, dbname)) as conn:
+        c = conn.cursor()
+        c.execute("DROP TABLE IF EXISTS dashboard_metrics")
```

Comment:
Same remark as in previous commit for monitoring: it's not obvious why this table is dropped (as is "last_measures" in monitoring part but *in contrast* with "metrics" there as well). A bit of explanation would be nice!

Reply:
I've put some explanations in bootstrap() docstring too